<launch>

	<include file="$(find p2os_driver)/launch/p2os_driver.launch" />
	
	<node pkg="p2os_teleop" type="p2os_teleop" name="p2os_teleop" respawn="true">
		<remap from="/des_vel" to="/move_base/cmd_vel" />
    <rosparam file="$(find p2os_teleop)/launch/p2os_teleop_ps3.yaml" command="load" />	
	</node>
	
	<node pkg="joy" type="joy_node" name="joy_node" />
</launch>
